core/spec.md

```diff
@@ -5546,6 +5588,42 @@ as defined below:
   - MUST NOT be present if the Resource's `hasdocument` model attribute is
     set to `false.
 
+##### `ancestor` Attribute
+- Type: String
+- Description: The `versionid` of this Version's ancestor.
+
+  The `ancestor` attribute MUST be set to any existing Version's `versionid`,
+  or to the `versionid` of the Version itself, to indicate that this Version
+  has no `ancestor`.
+
+  When creating a Version without explicitly setting the `ancestor`
+  attribute, the server MUST set the `ancestor` to the most recent Version's
+  `versionid` attribute, based on the `createdat` attribute. If multiple
+  exist, the lowest Version will be determined alphabetically. If no
```

Comment:
s/lowest/highest/   no?

And probably: If multiple exist, then the Version with the highest alphabetical `versionid` MUST be chosen.

Reply:
Let's say there is 1 root, with the 2 oldest versions that share the same date, one is A and one B. We'd want to remove A right? A is "lower" than B the way I see it, that's why I said "lowest"

Reply:
But you're not deleting, you're trying to find the newest so you can set the new version's ancestor, no?